build.sbt

```diff
@@ -78,6 +78,12 @@ Revolver.settings
 
 Revolver.enableDebugging(port = 5050, suspend = false)
 
+// When JAVA_OPTS are specified in the environment, they are usually meant for the application
+// itself rather than sbt, but they are not passed by default to the application, which is a forked
+// process. This passes them through to the "re-start" command, which is probably what a developer
+// would normally expect.
+javaOptions in Revolver.reStart := sys.env("JAVA_OPTS").split(" ")
+
```

Comment:
does this change the recommended way to start rawls, which is now `sbt "~reStart --- $JAVA_OPTS"`? If so, please update the readme in firecloud-develop.